src/test/app/XChain_test.cpp

```diff
@@ -631,10 +632,12 @@ struct XChain_test : public beast::unit_test::suite,
         env.tx(create_bridge(a1, bridge(a1, GUSD, B, BUSD))).close();
         env.tx(create_bridge(a2, bridge(a2, GUSD, B, BUSD))).close();
 
-        // Add the exact same bridge to two different accoutns (one must locking
-        // account and one must be issuing)
+        // Add the exact same bridge to two different accoutns (one locking
+        // account and one issuing)
         env.tx(create_bridge(a3, bridge(a3, GUSD, a4, a4["USD"]))).close();
-        env.tx(create_bridge(a4, bridge(a3, GUSD, a4, a4["USD"]))).close();
+        env.tx(create_bridge(a4, bridge(a3, GUSD, a4, a4["USD"])),
+               ter(tecDUPLICATE))
+            .close();
```

Comment:
Perhaps this should check the reverse situation compared to the above test change - issuing chain then locking chain.

Reply:
Fixed in `211ccc6fd9 [fold] Add additional test`